package.json

```diff
@@ -80,7 +80,7 @@
     "compressjs": "github:openpgpjs/compressjs",
     "elliptic": "github:openpgpjs/elliptic",
     "hash.js": "^1.1.3",
-    "node-fetch": "^1.7.3",
+    "node-fetch": "^2.1.2",
```

Comment:
The change here was required to have `arrayBuffer` function on fetch response objects. It is available only for node-fetch version >2.